[hi] Add managing-secret-using-kubectl.md

aryasoni98 opened the pull request:
### Description
This PR adds the Hindi translation of the "Managing Secrets using kubectl" documentation. The translation includes:
- Complete translation of the task guide for managing secrets using kubectl
- All code examples and commands preserved in their original form
- Technical terms maintained in English where appropriate
- Proper Hindi technical terminology used where applicable

This translation helps Hindi-speaking users understand how to manage secrets using the kubectl command-line tool.

### Issue
Closes: #50292

aryasoni98 committed:
Hindi translation: Add managing-secret-using-kubectl.md

aryasoni98 commented:
@divya-mohan0209 Can you please review my PR? Open to any feedback or suggestions!

divya-mohan0209 commented on content/hi/docs/tasks/configmap-secret/managing-secret-using-kubectl.md:
```diff
@@ -0,0 +1,187 @@
+
+## सीक्रेट बनाएं
+
+एक `Secret` ऑब्जेक्ट संवेदनशील डेटा जैसे क्रेडेंशियल्स को स्टोर करता है जिसका उपयोग पॉड्स द्वारा सेवाओं तक पहुंचने के लिए किया जाता है। उदाहरण के लिए, आपको डेटाबेस तक पहुंचने के लिए आवश्यक उपयोगकर्ता नाम और पासवर्ड को स्टोर करने के लिए एक सीक्रेट की आवश्यकता हो सकती है।
```
Secret needs to be transliterated.

aryasoni98 replied:
Fixed the transliteration of "Secret"

divya-mohan0209 commented on content/hi/docs/tasks/configmap-secret/managing-secret-using-kubectl.md:
````diff
@@ -0,0 +1,187 @@
+    --from-literal=username=admin \
+    --from-literal=password='S!B\*d$zDsb='
+```
+आपको अपनी स्ट्रिंग्स में विशेष वर्णों जैसे `$`, `\`, `*`, `=`, और `!` को एस्केप करने के लिए एकल उद्धरण `''` का उपयोग करना होगा। यदि आप ऐसा नहीं करते हैं, तो आपका शेल इन वर्णों की व्याख्या करेगा।
````
Can we translate escape here?

aryasoni98 replied:
@divya-mohan0209  Corrected the escape character explanation

divya-mohan0209 reviewed (commented):
Initial batch of reviews

aryasoni98 committed:
[hi] Fix Secret transliteration and escape character explanation in managing-secret-using-kubectl.md

aryasoni98 commented:
Hey @divya-mohan0209 @niranjandarshann 

Just a gentle reminder could you please review my PR when you get a chance? I'm open to any feedback or suggestions you might have. Thanks in advance!

divya-mohan0209 commented:
/retitle [hi] Add managing-secret-using-kubectl.md

aryasoni98 commented:
@divya-mohan0209 When you have a moment, could you please review it? I’m open to any suggestions or improvements you might have.

aryasoni98 commented:
@divya-mohan0209 @dipesh-rawat Hey, could you please review my PR? If everything looks good, feel free to go ahead and merge it. 

Thanks!

jayeshmahajan reviewed (approved):
looks good to me

aryasoni98 commented:
@divya-mohan0209 @jayeshmahajan Gentle reminder ,

Whenever you get a chance, please take a moment to review the PR. If everything looks good, kindly proceed with the merge. Appreciate your time and support!

divya-mohan0209 commented on content/hi/docs/tasks/configmap-secret/managing-secret-using-kubectl.md:
```diff
@@ -0,0 +1,187 @@
+
+<!-- steps -->
+
+## Secret बनाएं
```
Please transliterate Secret.

divya-mohan0209 reviewed (changes requested):
@aryasoni98 : Please transliterate the word, Secret.

aryasoni98 committed:
[hi] Add managing-secret-using-kubectl.md

aryasoni98 commented:
@divya-mohan0209 @niranjandarshann  @jayeshmahajan  I am merely reminding you about this PR. If everything looks excellent, kindly review and merge it whenever you get a chance. Thanks!

jayeshmahajan commented on content/hi/docs/tasks/configmap-secret/managing-secret-using-kubectl.md:
```diff
@@ -0,0 +1,188 @@
+
+### स्रोत फाइलों का उपयोग करें
+
+1. क्रेडेंशियल्स को फाइलों में स्टोर करें:
```
```suggestion
1. क्रेडेंशियल्स को फाइलों में संग्रह करें:
```

aryasoni98 committed:
[hi] Add managing-secret-using-kubectl.md

- Add Hindi translation of Managing Secrets using kubectl documentation
- Address review feedback: replace स्टोर with संग्रह throughout
- Transliterate Secret to सीक्रेट per localization guidelines
- Fix single quote notation: ''' to '' for escape character explanation
- Closes #50292

Made-with: Cursor

divya-mohan0209 commented on content/hi/docs/tasks/configmap-secret/managing-secret-using-kubectl.md:
```diff
@@ -0,0 +1,188 @@
+
+### स्रोत फाइलों का उपयोग करें
+
+1. क्रेडेंशियल्स को फाइलों में संग्रह करें:
```
Is the intention to transliterate conditional? In which case, the spelling is wrong.

divya-mohan0209 commented on content/hi/docs/tasks/configmap-secret/managing-secret-using-kubectl.md:
````diff
@@ -0,0 +1,188 @@
+   echo -n 'S!B\*d$zDsb=' > ./password.txt
+   ```
+
+   `-n` फ्लैग यह सुनिश्चित करता है कि जनरेट की गई फाइलों में टेक्स्ट के अंत में एक अतिरिक्त न्यूलाइन वर्ण नहीं है। यह महत्वपूर्ण है क्योंकि जब `kubectl` एक फाइल को पढ़ता है और सामग्री को base64 स्ट्रिंग में एनकोड करता है, तो अतिरिक्त न्यूलाइन वर्ण भी एनकोड हो जाता है। आपको फाइल में शामिल स्ट्रिंग्स में विशेष वर्णों को एस्केप करने की आवश्यकता नहीं है।
````
Please replace the transliterated generate with उत्पादित

divya-mohan0209 commented:
@aryasoni98 : Gentle reminder on the PR
